jmclient/jmclient/taker_utils.py

```diff
@@ -21,7 +21,7 @@
 Currently re-used by CLI script tumbler.py and joinmarket-qt
 """
 
-def direct_send(wallet_service, amount, mixdepth, destination, answeryes=False,
+def direct_send(wallet_service, amount, mixdepth, destination, answeryes=True,
```

Comment:
Don't change the default for `answeryes` to `True`, I think it's important that we don't default to automatic broadcast, but force callers to choose it.

Reply:
Done in https://github.com/abhishek0405/joinmarket-clientserver/commit/828cec29717e88f69cb82c20fc42b3699199d9f8